client/src/modules/CrossCheck/components/criteria/SubtaskCriteria.tsx

```diff
@@ -1,76 +1,79 @@
-import { Input, Rate, Typography } from 'antd';
+import { Input, Typography, InputNumber, Slider } from 'antd';
+import { useMemo } from 'react';
 import { CrossCheckCriteriaData } from 'services/course';
 
-import { CommentState, CountState } from '../CrossCheckCriteriaForm';
-
 const { TextArea } = Input;
+const { Text } = Typography;
+
 interface SubtaskCriteriaProps {
-  criteriaComment: CommentState[];
   subtaskData: CrossCheckCriteriaData;
-  countStar: CountState[];
-  updateCountStar: (event: number, max: number, key: string) => void;
-  updateComment: (value: string, key: string) => void;
+  updateCriteriaData: (updatedEntry: CrossCheckCriteriaData) => void;
 }
 
-const NUMBER_OF_STARS = 5;
+export default function SubtaskCriteria({ subtaskData, updateCriteriaData }: SubtaskCriteriaProps) {
+  const maxScore = subtaskData.max as number;
+  const comment = subtaskData.textComment as string;
+  const criteriaScore = subtaskData.point as number;
```

Comment:
Plesase consider to avoid these typecasts somehow ( e.g. use typeguard/fallback on default values)